docs/README.md

```diff
@@ -72,43 +68,65 @@ Functionally, the app has several key features:
 
 [More about how/why to interact with web3 here](https://gitcoin.co/web3).
 
-Technically, the system is architected:
+## Technically
+
+The system is architected:
 
 * __Web3__ The main source of truth for the system is the Ethereum blockchain. Check out the [smart contracts](https://github.com/gitcoinco/smart_contracts).
 * __Web2__ This part of the app is built with Python, Django, Postgres, and a handful of other tools that are common in the web2 ecosystem.
 * __Web 3 Bridge__ This is the bridge between web3 and the rest of the application. Mostly built in javascript(web3js) and python(web3py).
 * __Brochureware__ Just a nice little landing page telling folks what the Gitcoin project is.
 
+
+## How to interact with this repo
+
+### On Github
+
+[Star](https://github.com/gitcoinco/web/stargazers) and [watch](https://github.com/gitcoinco/web/watchers) this github repository to stay up to date, we're pushing new code several times per week!
+
+Check out the [CHANGELOG](https://docs.gitcoin.co/mk_changelog/) for details about recent changes to this repository.
```

Comment:
Could we replace this with releases https://github.com/gitcoinco/web/releases ?
Our changelog is captured there

Reply:
changed